src/axom/core/execution/runtime_policy.hpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef AXOM_CORE_EXECUTION_RUNTIME_POLICY_HPP_
+#define AXOM_CORE_EXECUTION_RUNTIME_POLICY_HPP_
+
+#include "axom/config.hpp"   /* for compile time defs. */
+#include "axom/fmt/format.h" /* for axom::fmt */
+
+#include <map>
+
```

Comment:
It might be a good idea to add a `\file` comment describing the intent of these enums and macros